tests/subresource_api_test.go

```diff
@@ -161,15 +161,13 @@ var _ = Describe("[sig-compute]Subresource Api", decorators.SigCompute, func() {
 	Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:component] VirtualMachine subresource", func() {
 		Context("with a restart endpoint", func() {
 			It("[test_id:1304] should restart a VM", func() {
-				vm := libvmi.NewVirtualMachine(libvmifact.NewGuestless())
+				vm := libvmi.NewVirtualMachine(libvmifact.NewGuestless(), libvmi.WithRunStrategy(v1.RunStrategyAlways))
 				vm, err := virtCli.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
 				Expect(err).NotTo(HaveOccurred())
 
-				tests.StartVirtualMachine(vm)
+				Eventually(matcher.ThisVM(vm)).WithTimeout(300 * time.Second).WithPolling(time.Second).Should(matcher.BeReady())
 				vmi, err := virtCli.VirtualMachineInstance(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
 				Expect(err).NotTo(HaveOccurred())
-				Expect(vmi.Status.Phase).To(Equal(v1.Running))
```

Comment:
Why was this removed?

Reply:
https://github.com/kubevirt/kubevirt/blob/ec14523ae05a7053516b0a8bd2e229cf9d96500c/staging/src/kubevirt.io/api/core/v1/types.go#L1625

IIUC we're waiting for it to be ready, it means it's already running, so there's no need to check it again.

Reply:
@Barakmor1 
The cost of the check is limit to zero. On the other hand if we ever break the API contract this check would catch it. Consider re-adding it.